Peggle/Peggle.lua

```diff
@@ -69,11 +74,10 @@ local t = function(...)
-};
 e.GetBackdrop = function()
 	return{bgFile = "Interface\\Tooltips\\UI-Tooltip-Background", tileSize = 16, edgeFile = "Interface\\Tooltips\\UI-Tooltip-Border", tile = 1, edgeSize = 16, insets = {top = 5, right = 5, left = 5, bottom = 5, }}
 end
+C_ChatInfo.RegisterAddonMessagePrefix(e.addonName);
```

Comment:
Register Addon prefix to allow it to send messages on the hidden addon channel (to fix duel/battle)